compiler/quilt/tools/util.py

```diff
@@ -81,3 +81,20 @@ def gzip_compress(data):
     with gzip.GzipFile(fileobj=buf, mode='wb') as fd:
         fd.write(data)
     return buf.getvalue()
+
+def children(path, dirs=True, files=True, noinvisible=True):
+    """
+    Return children of a given path
+    dirs=True => include dirs
+    files=True => include files
+    invisible=True => include files that do begin with .
+    """
+    matches = os.listdir(path)
+    if noinvisible:
+        matches = filter(lambda x: not x.startswith('.'), matches)
```

Comment:
Actually, list comprehension - `[x for x in ... if ...]` - is preferred over `filter` and `lambda`.